app/models/declaration/wcodeclarationModels.scala

```diff
@@ -16,6 +16,7 @@
 
 package models.declaration
 
+//FIXME deadcode
 case class BorderTransportMeans(
```

Comment:
It's wcodecalrationModels, so probably it's unused. In this case please remove it